app/templates/recommendation.html

```diff
@@ -0,0 +1,25 @@
+{% extends "base.html" %}
+
+{% block content %}
+<h1>Welcome to Song Recommendation!</h1>
+<form action="" method="post">
+    Select Genre:
+    <select name="" id="">
+        <option>Country</option>
+        <option>Rock</option>
+        <option>Pop</option>
+        <option>Latin</option>
+    </select>
+    <br>
+    <button type="submit">Get Recommendation</button>
+</form>
+
+<div class="recommended-songs">
+    <h1>Recommended Songs:</h1>
+    <ul>
+        <li>Blinding Lights</li>
+        <li>Blinding Lights</li>
+        <li>Blinding Lights</li>
+    </ul>
+</div>
+{% endblock %}
```

Comment:
This looks fine to me! 

nit: use different examples in the recommended song list to make it clearer that those are separate recommendations

Reply:
nit: add new line character at the end of line 25